composer.json

```diff
@@ -52,6 +59,12 @@
     ],
     "lint": [
       "@php ./vendor/php-parallel-lint/php-parallel-lint/parallel-lint . -e php --exclude vendor --exclude .git"
+    ],
+    "test": [
+      "@php ./vendor/phpunit/phpunit/phpunit --no-coverage"
```

Comment:
Using `phpunit` here is generally a bad idea as if there is a `phpunit` executable in the system path before the path to Composer, that will often take precedence.

Using `vendor/bin/phpunit` is also not a good idea as then the PHP version used by Composer won't be respected.

Using `@php vendor/bin/phpunit` would work for *nix systems, but breaks on Windows.

And that's how you end up with `@php ./vendor/phpunit/phpunit/phpunit` which:
* Respects the PHP version used by Composer
* Will always use the Composer installed version of PHPUnit rather than whatever system default is set up
* Will work cross-platform.

Reply:
Cool. Thanks for the details!